cpp/test/samples/evm-operations.js

```diff
@@ -0,0 +1,44 @@
+const { toBeHex, zeroPadValue, parseUnits } = require('ethers')
+const { no0x } = require('../utils/wharfkit-ext')
+
+const amounts = {
+  pegin: '5.87190615',
+  peginWithUserData: '0.099924825',
+}
+
```

Comment:
I think these are better to be computed in place rather than hardcoded here (see comment where this is used)